pkg/stream/coordinator.go

```diff
@@ -132,7 +132,7 @@ func newResponse(commandDescription string) *Response {
 	res := &Response{}
 	res.commandDescription = commandDescription
 	res.code = make(chan Code, 1)
-	res.data = make(chan interface{})
+	res.data = make(chan interface{}, 1)
```

Comment:
Why here is it necessary to add `1` to the channel?

Reply:
It was done because the [test failed](https://github.com/rabbitmq/rabbitmq-stream-go-client/tree/f323ce5e87c4ad4b3321ef323f98130e29400318/pkg/stream/consumer_test.go#L325-L333). I found that if there is no offset for this consumer on the server the `OffsetNotFoundError` would be raised from `queryOffset` ([the error code - `responseCodeNoOffset` is send from queryOffsetFrameHandler](https://github.com/rabbitmq/rabbitmq-stream-go-client/tree/f323ce5e87c4ad4b3321ef323f98130e29400318/pkg/stream/server_frame.go#L431)), but a bit later the [0 offset would be send to the non buffered `data` chanel](https://github.com/rabbitmq/rabbitmq-stream-go-client/tree/f323ce5e87c4ad4b3321ef323f98130e29400318/pkg/stream/server_frame.go#L431) thus the `queryOffsetFrameHandler` hangs until `queryOffset` would read from channel (but this function already exited - there is no readers) as a result `Client.handleResponse` loop hangs too. So I've made data channel buffered, like it's done for the code channel.